src/schemathesis/runner/impl/core.py

```diff
@@ -271,16 +295,30 @@ def wsgi_test(
 ) -> None:
     # pylint: disable=too-many-arguments
     headers = _prepare_wsgi_headers(headers, auth, auth_type)
+    _wsgi_test(case, checks, targets, result, headers, store_interactions, feedback)
+    add_cases(case, _wsgi_test, checks, targets, result, headers, store_interactions, feedback)
+
+
+def _wsgi_test(
+    _case: Case,
```

Comment:
And in this case, as well - there is no shadowing of another `case` variable, so we can keep it without the leading underscore